web/src/client/network/network_manager.js

```diff
@@ -24,22 +24,25 @@
 import { DBusClient } from "../dbus";
 import cockpit from "../../lib/cockpit";
 import { intToIPString, stringToIPInt } from "./utils";
-import { NetworkEventTypes, ConnectionState } from "./index";
+import { NetworkEventTypes } from "./index";
 import { createAccessPoint, SecurityProtocols } from "./model";
 
 /**
  * @typedef {import("./model").Connection} Connection
  * @typedef {import("./model").ActiveConnection} ActiveConnection
  * @typedef {import("./model").IPAddress} IPAddress
  * @typedef {import("./model").AccessPoint} AccessPoint
+ * @typedef {import("./index").NetworkEventFn} NetworkEventFn
  */
 
-const NM_SERVICE_NAME = "org.freedesktop.NetworkManager";
-const NM_IFACE = "org.freedesktop.NetworkManager";
-const NM_SETTINGS_IFACE = "org.freedesktop.NetworkManager.Settings";
-const NM_CONNECTION_IFACE = "org.freedesktop.NetworkManager.Settings.Connection";
-const NM_ACTIVE_CONNECTION_IFACE = "org.freedesktop.NetworkManager.Connection.Active";
-const NM_IP4CONFIG_IFACE = "org.freedesktop.NetworkManager.IP4Config";
+const SERVICE_NAME = "org.freedesktop.NetworkManager";
+const IFACE = "org.freedesktop.NetworkManager";
+const SETTINGS_IFACE = "org.freedesktop.NetworkManager.Settings";
+const CONNECTION_IFACE = "org.freedesktop.NetworkManager.Settings.Connection";
+const ACTIVE_CONNECTION_IFACE = "org.freedesktop.NetworkManager.Connection.Active";
```

Comment:
NP: if using "AP_IFACE" instead "ACCESS_POINTS_IFACE"… then use "AC_IFACE" instead "ACTIVE_CONNECTION_IFACE". For consistence. That said, I prefer reading longer versions. Just a neat-picking.